### DISCUSSION TITLE
Bypass consent page

### DISCUSSION THREAD

**Discussion by bandirsen** [babelouest/glewlwyd#190] [Q&A] [answered]
Sometimes for trusted/internal client app, we don't want to display consent to end-user, is there any configuration skip/bypass consent page for certain client ?

**Comment by babelouest** [comment, marked as answer]
You can skip the consent page by adding the query parameter `g_continue` to the `/auth` endpoint.

If the user session is valid and has the expected credentials, the consent page will be automatically skipped.

If the user session isn't valid or the password has expired or no scopes have been granted for example, then the consent page will be displayed anyway.

The url would look like that:

https://glewlwyd.tld/api/oidc/auth?response_type=code&scope=openid&nonce=abcd&state=xyz&client_id=xxx&redirect_uri=https://example.com&g_continue

**Reply by bandirsen** [reply]
Thanks you
